biojava-structure/src/main/java/org/biojava/nbio/structure/Structure.java

```diff
@@ -788,4 +765,53 @@
 	 *  returned the same value as {@link #getPDBCode()}
 	 */
 	String getIdentifier();
+	
+	/**
+	 * Gets the keywords (KEYWODS) record of the structure
+	 * @return The keywords in a <code>List&lt;String&gt;</code>
+	 * @since 6.0.0
+	 */
+	List<String> getKeywords();
+	
+	/**
+	 * Sets the KEYWODS record of the structure.
+	 * @param keywords The keywords in a <code>List&lt;String&gt; to set.</code>
+	 * @since 6.0.0
+	 */
+	void setKeywords(List<String> keywords);
+
+	/**
+	 * Get PDB code of structure.
+	 *
+	 * @return a String representing the PDBCode value
+	 * @see #setPDBCode
+	 * @deprecated use {@link #getPDBId()} to get a {@link PDBId} object or getPDBId().getId() to get a {@link String}
+	 */
+	String getPDBCode () ;
+
+	/**
+	 * Set PDB code of structure .
+	 *
+	 * @param pdb_id  a String specifying the PDBCode
+	 * @throws PDBIdException 
+	 * @see #getPDBCode
+	 * @deprecated use {@link #setPDBCode(PDBId)}
+	 */
+	void setPDBCode (String pdb_id) throws PDBIdException ;
+
+//	/**
+//	 * Returns the PDB identifier associated with this StructureIdentifier.
+//	 * @deprecated From BioJava 4.2, use {@link #getPDBCode()} or
+//	 *  <code>getStructureIdentifier().toCanonical().getPdbId()</code>
+//	 */
+//	@Deprecated
+//	String getPdbId();
+//
+//	@Deprecated
+//	void setPdbId(String pdb_id);
+
+
+	PDBId getPDBId();
```

Comment:
Needs docstring

Reply:
Docstring should mention that `getPDBId().getId()` replaces the old (differently capitallized) version.

Reply:
I will add the documentation soon. We just need to catch the new release 6.0.0 first.